assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/useCellIcon.tsx

```diff
@@ -20,7 +20,7 @@ const useCellIcon = (): Callback => {
 				case 'REMOVED':
 					return () => <CloseOutlined fill='white' />;
 				default:
-					return () => <MinusOutlined fill='white' />;
+					return () => <MinusOutlined />;
```

Comment:
it was added here, before: https://github.com/eventespresso/event-espresso-core/pull/2729#discussion_r414701253

Reply:
now:

![image](https://user-images.githubusercontent.com/1477453/81303288-2dff2480-9084-11ea-9604-d71b620bedb8.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/81303329-39eae680-9084-11ea-9b50-1d6e833d33f2.png)